data/json/items/comestibles/carnivore.json

```diff
@@ -1898,5 +1898,58 @@
     "fun": 6,
     "weight": "298 g",
     "flags": [ "EATEN_HOT" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "meat_mi-go",
+    "category": "other",
+    "name": { "str": "chunk of mi-go meat", "str_pl": "chunks of mi-go meat" },
+    "weight": "238 g",
+    "color": "pink",
+    "spoils_in": "90 days",
+    "//": "It's so alien few micro organisms are able to degrade it.",
+    "use_action": [ "POISON" ],
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 0,
+    "description": "A piece of mi-go meat that's obviously alien, given its pink color, odd odor, and strange texture.  It would not be advisable to try to eat it.",
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "mi-go_flesh" ],
+    "volume": "250 ml",
+    "fun": -10,
+    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE" ]
+  },
+  {
+    "id": "blood_mi-go",
+    "type": "COMESTIBLE",
+    "copy-from": "meat_mi-go",
+    "name": { "str_sp": "mi-go blood" },
+    "description": "Mi-go blood that's obviously alien, given its pink color and strange odor.  It would not be advisable to try to eat it.",
```

Comment:
```suggestion
    "description": "Goo from inside a mi-go, perhaps some kind of blood?  It's obviously alien, given its pink color and strange, nostril-stinging odor.  It would not be advisable to try to eat it.",
```